Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -282,7 +260,6 @@ public static void UnregisterSpawnHandler(Guid assetId)
 
         public static void ClearSpawners()
         {
-            prefabs.Clear();
```

Comment:
why is this not needed anymore?

Reply:
Because the prefabs hash set does not do anything anymore.   I could remove it instead of obsolete it.